pkg/kritis/metadata/metadata.go

```diff
@@ -48,4 +48,5 @@ type Vulnerability struct {
 type PGPAttestation struct {
 	Signature string
 	KeyID     string
+	OccName   string
```

Comment:
I don't know what an OccName is. Do you mind adding a docstring so that this shows up in our autogenerated docs properly?